### ISSUE TITLE
Field Strength Displays Different Values

### ISSUE THREAD

**Issue by StalkerM8** [acidjazz/drmon#3]
Not sure what is causing this but the Field Strength displays a few numbers below what it should.

http://puu.sh/sJh10/0c80530536.png -Example of it doing this.

http://puu.sh/sJh5T/fde3a99ba9.png -Example of it being at 50%

http://puu.sh/sJhnf/adc2daec72.png -Heres a better view

This is really weird and i don't code so.... an explanation for this would be good. Thanks

**Comment by acidjazz** [comment]
@piepie62 has created a [pull request](https://github.com/acidjazz/drmon/pull/4) to fix this which I've tested and merged, go ahead and [upgrade](https://github.com/acidjazz/drmon#upgrading-to-the-latest-version) and you should be good

**Comment by piepie62** [comment]
Actually, my PR does not fix this, this is due to the fact that for some reason his max containment field strength is less than the one you assumed (and I have, I'm actually not quite sure why his is 95M, but whatever). I will fix it right now, though.

**Comment by StalkerM8** [comment]
So this is still broken? Still showing all this lower than it should and its now doing this for the energy saturation as well.

After-note: this reactor is setup in the End and the server does shut down regularly dont know if that helps, but it is also in Infinity Evolved 2.4.2

**Comment by piepie62** [comment]
Right now the install script is not updated to my edited version. It will not update to it until @acidjazz  accepts the [pull request](https://github.com/acidjazz/drmon/pull/5), if he does. The fuel should be lower than before; you had it somewhere very close to 100 percent before, which was normal before my fix and it should be less now. Could you send a screenshot of your energy saturation, as well? I could check to make sure it is correct if you do. If there is something wrong, I'll look into it.

**Comment by piepie62** [comment]
Also, the location and frequency of server stops shouldn't have any affect - I say shouldn't because weird things do happen. It may have something to do with chunk boundaries; is it chunkloaded? Mine did weird things because the screen was halfway in a different chunk.

**Comment by piepie62** [comment]
Oh, wait, the energy saturation? Whoops, misunderstood you. I can fix that; that must have been happening before this anyway.

EDIT: Fixed in the latest commit to #5

**Comment by StalkerM8** [comment]
Yes it is chunkload via the Railcraft personal loader cause you can't claim chunks in the End via the FTB Utilities and yes the setup its in it take up two different chunks, as for the energy saturation that will be a bit cause i have no idea if my freind will feel like putting it up.

PS:The moniter flickers a lot and its rather hurtful on the eyes.

**Comment by piepie62** [comment]
Actually, I don't need to see it, I could fix it without seeing it. (At least I think I could; I can't tell for certain without seeing it, but my fix does make sense.)

**Comment by StalkerM8** [comment]
Well Soon however I will be moving it to the Last Millennium cause there i dont need that chunk loader instead i can use the FTB Utils, but if you manage to fix this its much appreciated

**Comment by piepie62** [comment]
I think it should be fixed with the [pull request](https://github.com/acidjazz/drmon/pull/5).

**Comment by piepie62** [comment]
And I'm not sure what the monitor flickering is; I don't think even @acidjazz does.

**Comment by StalkerM8** [comment]
Ive read some places its the fact the screen is displaying colors and that is tied in with the refresh rate so to speak, idk either sometimes its good sometimes it literally having a seizure

**Comment by StalkerM8** [comment]
http://puu.sh/sNfso/9d8f9275dc.png 
http://puu.sh/sNfx8/46f336ceec.png
Dont mind the PorjectRed Stuff thats just a bit that my one friend added for singals to display the state the reactor is in

**Comment by piepie62** [comment]
Yeah, that looks fine. I just think that the update needs to be pushed to the main repository and then it'll work.

**Comment by StalkerM8** [comment]
This is probably annoying at this point but now the reactor is at 100 mil for field strength and 1 bil for energy sat and its displaying a few decimals above the norm
http://puu.sh/sOSVX/a11030ea1c.png

Also noticing now an unknown little thing where when the output of RF gets to the target set, then Energy sat. shoots up a moment making the power fluctuate a couple 100

**Comment by acidjazz** [comment]
@StalkerM8 @piepie62 i re-opened this issue and merged pull request #5 , let me know if this fixes things

**Comment by StalkerM8** [comment]
ALrighty ill post anything else unusual

**Comment by StalkerM8** [comment]
After last night the program still shows the odd decimal (i didn't break the computer i simply deleted both startup and install then re did the pastebin) and it still doing it but im fine as long as its displaying temperature right and the safety stuff is right, otherwise my only difficulty so far is the rapid blinking of the monitor.

Little update i thought id give to you after a day, need anything else you know what to do :P

**Comment by piepie62** [comment]
Hmm... how far off is it? The problem you had before was an error in the logic of the program, but is there a possibility that this is a small amount of roundoff error? If it is, then the percentage of the field strength on the screen should be just a little bit over the strength in the reactor GUI, due to LUA not having a round function, and, therefore, the usage of rounding up in all cases. If not, could you detail it a bit more?
EDIT: Looking at the picture from three days ago, that is a little bit more than roundoff error, but still should be okay. Is it still a .02% error, less or more? More is a problem, less is ideal, and the same is slightly vexing.
The safety stuff should work well enough right now; .02% shouldn't affect much. Then again, it could be a difference of life and death in the event of an explosion.

**Comment by StalkerM8** [comment]
If i remember from seeing it its both Energy Sat. and Field Strength, and looking at the GUI its 50% nearly so thats fine but then it arises to like 50.11% something above 50 but not 51 never have I seen it above above 51 when its been running (unless I recently shut it down) That is interesting though with the rounding. As for the safety stuff its worked fine, It almost exploded on me cause the power source for it got unloaded (reached 2.9% while trying to stop the reactor luckily) I'll take a look for you when im able to. 

Again this probably annoying for you but you know, simply giving feedback :P

**Comment by piepie62** [comment]
Oh, don't worry, the only thing that annoys me about this is that I can't see a logical reason for it to act like this. It just seems... odd.

**Comment by piepie62** [comment]
Hmmm... I am noticing this now as well on the field strength. I just don't understand it... It should not be doing this...

**Comment by StalkerM8** [comment]
So its still doing this weird decimal...Checked in after a long while updated it on the Computer by deleting the entire thing and still showing 50.11%
